docs/usage/self-hosting.md

```diff
@@ -144,6 +144,38 @@ spec:
               emptyDir: {}
 ```
 
+#### CircleCI
+
+If you are using CircleCI, you can use the third-party [daniel-shuy/renovate](https://circleci.com/developer/orbs/orb/daniel-shuy/renovate) orb to run a self-hosted instance of Renovate on CircleCI.
+
+By default, the orb looks for the self-hosted configuration file in the project root, but can be customized using the `config_file_path` parameter.
```

Comment:
```suggestion
By default, the orb looks for the self-hosted configuration file in the project root, but you can specify another path to the configuration file with the `config_file_path` parameter.
```

What do you think of this?

Reply:
Thanks, I think I'll actually apply the same improvement to the orb documentation as well!